crates/globset/src/lib.rs

```diff
@@ -16,6 +16,19 @@ globs as matching.
 
 This example shows how to match a single glob against a single file path.
 
+
+# Crate features
```

Comment:
Why is this added in the middle of an example? It should go at the bottom of the crate docs.